src/Translation_unit.ml

```diff
@@ -316,26 +320,36 @@ let parse_print (XUnit xunit) (conf : Conf.t) ~input_name ~input_file ic
           Format.eprintf "  BUG: %s.\n%!" s ;
           ( match m with
           | `Doc_comment l when not conf.Conf.quiet ->
-              List.iter l ~f:(fun (loc_before, loc_after, msg) ->
-                  if Location.compare loc_before Location.none = 0 then
+              List.iter l ~f:(function
+                | `Moved (loc_before, loc_after, msg) ->
+                    if Location.compare loc_before Location.none = 0 then
+                      Caml.Format.eprintf
+                        "%!@{<loc>%a@}:@,@{<error>Error@}: Docstring (** \
+                         %s *) added.\n\
+                         %!"
+                        Location.print_loc loc_after (String.strip msg)
+                    else if Location.compare loc_after Location.none = 0
+                    then
+                      Caml.Format.eprintf
+                        "%!@{<loc>%a@}:@,@{<error>Error@}: Docstring (** \
+                         %s *) dropped.\n\
+                         %!"
+                        Location.print_loc loc_before (String.strip msg)
+                    else
+                      Caml.Format.eprintf
+                        "%!@{<loc>%a@}:@,@{<error>Error@}: Docstring (** \
+                         %s *) moved to @{<loc>%a@}.\n\
+                         %!"
+                        Location.print_loc loc_before (String.strip msg)
+                        Location.print_loc loc_after
+                | `Unstable (loc, s) ->
                     Caml.Format.eprintf
-                      "%!@{<loc>%a@}:@,@{<error>Error@}: Docstring (** %s \
-                       *) added.\n\
+                      "%!@{<loc>%a@}:@,@{<error>Error@}: Formatting of (** \
+                       %s *) is unstable, please tighten up this comment \
```

Comment:
Maybe add after `is unstable` something like `(e.g. parses as a list or not depending on the margin)` to give users a better idea of what is meant by 'unstable' and what to look out for.